app/src/app/content-plugin/components/header/header.component.ts

```diff
@@ -42,11 +42,18 @@ import { isContentServiceEnabled } from '@alfresco/aca-shared/rules';
 })
 export class AppHeaderComponent implements OnInit, OnDestroy {
   private onDestroy$: Subject<boolean> = new Subject<boolean>();
+
   @Output()
   toggleClicked = new EventEmitter();
```

Comment:
you have removed the code that invokes this emitter, it should be cleaned in case nothing outside uses it